contrib/windows/juliarc.jl

```diff
@@ -1,6 +1,11 @@
 # This file is a part of Julia. License is MIT: http://julialang.org/license
 
 let user_data_dir
+    try
+        run(`stty`)
+    catch
+        ENV["PATH"] = joinpath(JULIA_HOME,"..","mintty","usr","bin")*";"*ENV["PATH"]
+    end
```

Comment:
I only conditionally add the `mintty` path when `stty` can't otherwise be found. I was worried about the case of starting julia from within `msys2` or `cygwin`. I didn't want to shadow the `stty` that would be available in that case, just in case it would do bad things, but I didn't verify that it would be an issue, either, so this extra bit of complication may be unnecessary.